### ISSUE TITLE
IGC connector build failing

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#1379]
```
12:29:55 [INFO] Egeria Connector for IBM Information Server - packaging 1.1 SKIPPED
12:29:55 [INFO] ------------------------------------------------------------------------
12:29:55 [INFO] BUILD FAILURE
12:29:55 [INFO] ------------------------------------------------------------------------
12:29:55 [INFO] Total time: 43.578 s
12:29:55 [INFO] Finished at: 2019-08-14T11:29:55Z
12:29:55 [INFO] ------------------------------------------------------------------------
12:29:55 [ERROR] Failed to execute goal on project ibm-igc-rest-client-library: Could not resolve dependencies for project org.odpi.egeria:ibm-igc-rest-client-library:jar:1.1: Could not find artifact org.odpi.egeria:http-helper:jar:1.1 in releases (https://nexus.odpi.org/content/repositories/releases/) -> [Help 1]
12:29:55 org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project ibm-igc-rest-client-library: Could not resolve dependencies for project org.odpi.egeria:ibm-igc-rest-client-library:jar:1.1: Could not find artifact org.odpi.egeria:http-helper:jar:1.1 in releases (https://nexus.odpi.org/content/repositories/releases/)
12:29:55     at org.apache.maven.lifecycle.internal.LifecycleDependencyResolver.getDependencies (LifecycleDependencyResolver.java:269)
12:29:55     at org.apache.maven.lifecycle.internal.LifecycleDependencyResolver.resolveProjectDependencies (LifecycleDependencyResolver.java:147)
```
IGC connector staging build failing

**Comment by cmgrote** [comment]
Seems to be that for some reason the build process is now looking for dependencies under released versions of Egeria, rather than snapshots:

`org.odpi.egeria:http-helper:jar:1.1 in releases (https://nexus.odpi.org/content/repositories/releases/)`

Any idea why that's the case?  My root-level pom's still refer to `1.1-SNAPSHOT` as the version for the dependency, not `1.1`.

**Comment by planetf1** [comment]
The staging builds automatically remove the -SNAPSHOT designator. (merge builds do not) . Their intent is they are what would/could eventually *really* be released. It's part of the current LF release process. 

We may need to work with the LF build team (jira?) to figure out how to address properly. In egeria's case we only use external released dependencies or intra-repo dependencies. For your igc repo you also have a dependency on unreleased egeria artificats.. Hmm.

**Comment by planetf1** [comment]
@schannamallu Any advice on this? Do we perhaps add the nexus staging repo for egeria to this build - somehow marking this project as dependent on the egeria repo build?

**Comment by cmgrote** [comment]
(Just noting as well that whatever we do to fix this here will also apply to #1380)

**Comment by cmgrote** [comment]
Closing in light of the Azure pipeline migration (this seems stale) -- suggest opening with new error if there is still an issue in this realm in Azure.
